packages/python-packages/apiview-stub-generator/tests/docstring_parser_test.py

```diff
@@ -217,3 +222,33 @@ def test_docstring_defaults_legacy(self):
         assert parser.default_for("value") == "cat"
         assert parser.default_for("another") == "dog"
         assert parser.default_for("some_class") == ":py:class:`apistubgen.test.models.FakeObject`"
+
+    def test_docstring_complex_argtype_ivar(self):
+        obj = DocstringClass
+        class_node = ClassNode(name=obj.__name__, namespace=obj.__name__, parent_node=None, obj=obj, pkg_root_namespace=self.pkg_namespace)
+        actuals = _render_lines(_tokenize(class_node))
+        expected = [
+            "class DocstringClass:",
+            "ivar name: str",
+            "ivar values: Optional[dict[str, str]]", # TODO: Should Dict vs. dict be mixed in like this?
+        ]
+        _check_all(actuals, expected, obj)
+
+        expected_init = [
+            "def __init__(",
+            "self, ",
+            "name: Optional[str], ",
+            "*args: Any, ",
+            "*, ", # TODO: Second asterisk should be removed
```

Comment:
Are you planning this for this PR?

Reply:
I was planning on doing this is a separate PR since it's an unrelated change.